priv/repo/migrations/20200730155900_add_invite_to_hub_entry_mode_enum.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Ret.Repo.Migrations.AddInviteToHubEntryModeEnum do
+  use Ecto.Migration
+  @disable_ddl_transaction true
+
+  def change do
+    Ecto.Migration.execute("ALTER TYPE ret0.hub_entry_mode ADD VALUE IF NOT EXISTS 'invite'")
```

Comment:
Don't think you are allowed to use an `execute` in a `change` migration, since it can't automatically generate `up` and `down` for you. This seems pretty easy to reverse so I would just manually write the up/down migrations.

Reply:
So it turns out that removing a value from a type is pretty complicated, not to mention it's not obvious what we should do with existing data in this scenario. If it were easy, I would have made a change here, but generally I would like us to understand if/why reversible migrations are important to us, because I'm not totally convinced that they are worth the trouble in our typical migration patterns.